apps/api/src/app/events/usecases/trigger-event/trigger-event.usecase.ts

```diff
@@ -153,4 +156,39 @@ export class TriggerEvent {
       // eslint-disable-next-line no-console
       .catch((e) => console.error(e));
```

Comment:
Hey @scopsy 
small question f you don't mind it's not related to this PR, just noticed this legacy code line, is there was any reason for the catch and cosnole.log here?

Reply:
Mainly because didn't want to fail the trigger event because of it, but maybe better to a Sentry.captureExcepction here aswell. Wdyt?

Reply:
Yea that's the reason I thought the writer was aiming to.